cargo-dist/src/cli.rs

```diff
@@ -324,6 +328,14 @@ pub struct LinkageArgs {
 #[derive(Args, Clone, Debug)]
 pub struct HelpMarkdownArgs {}
 
+#[derive(Args, Clone, Debug)]
+pub struct UpdateArgs {
+    /// Run `cargo dist init` after performing an upgrade
+    #[clap(long)]
+    #[clap(default_value_t = true)]
+    pub run_init: bool,
```

Comment:
the polarity of this seems like messed up UX for the user. `--run-init=false`?

Reply:
Right, you want to make this `skip_init` instead of `run_init=false`?

Reply:
yeah

Reply:
Cool, will do